ledger/ledger-api-auth/src/test/suite/scala/com/digitalasset/ledger/api/auth/AuthServiceJWTCodecSpec.scala

```diff
@@ -62,9 +70,12 @@ class AuthServiceJWTCodecSpec
       "work for arbitrary standard Daml token values" in forAll(
         Gen.resultOf(StandardJWTPayload),
         minSuccessful(100),
-      )(value => {
-        serializeAndParse(value) shouldBe Success(value)
-      })
+      ) { value =>
+        // `participantId` is being used to identify the format of the token `ParticipantId` or `Scope`.
```

Comment:
I don’t understand what you’re trying to do here. Shouldn’t we be able to distinguish based on the prefix rather than whether the participant id is an empty string or not?